src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -1215,4 +1214,40 @@ public void viewAllRepliesInCommentsDrawer(String comment) {
     getCommentsDrawerViewAllReplies(comment).clickOnElement();
   }
 
+  private BaseElementFacade getReactionActivityList(String activity) {
+    return findByXPathOrCSS(String
+            .format("(//*[contains(text(),'%s')]//following::p[contains(@class,'likersNumber')])[1]", activity));
```

Comment:
1. Why using `[1]`? Does this xPath expression returns more than one element ?
Please avoid using `following::` because it's not precise at all. for this particular expression, I suggest : 

```suggestion
            .format("//*[contains(text(),'%s')]//ancestor::*[contains(@class, 'activity-detail')]//*[contains(@class,'likersNumber')]", activity));
```

This expression will return one exact single match for a given activity.
2. It's better to not use exact element name (like `p` here) in an xPath only when it's meant to be a `p` and that the Automatic Test case has to fail when it changes.

Reply:
xpath changed